### PR TITLE
Add Python API to tutos

### PR DESCRIPTION
**Pull request by FlorianJacta**

### CODE REVIEW

File: docs/tutorials/articles/3rd_party_components/index.md [hunk 1]
```diff
@@ -93,9 +93,20 @@ Gui.register_content_provider(Map, expose_folium)
...
+    <|part|content={folium_map()}|>
+    ```
+=== "Python"
+    ```python
+    import taipy.gui.builder as tgb
+
+    with tgb.Page() as page:
+        ...
+        tgb.part(content="{folium_map()}")
+    ```
```
Comment (jrobinAV) [inline docs/tutorials/articles/3rd_party_components/index.md:108]: Don't we want to put Python first? I thought that was what our users preferred.

File: docs/tutorials/articles/3rd_party_components/src/example.py [hunk 2]
```diff
@@ -41,4 +41,11 @@ def folium_map():
 
     
 Gui.register_content_provider(Map, expose_folium)
+
+# import taipy.gui.builder as tgb
+
+# with tgb.Page() as page:
+#     ...
+#     tgb.part(content="{folium_map()}")
+
```
Comment (jrobinAV) [inline docs/tutorials/articles/3rd_party_components/src/example.py:50]: To be removed?

File: docs/tutorials/articles/colab_with_ngrok/index.md [hunk 3]
```diff
@@ -115,14 +115,21 @@ To address this issue, you can modify the *change_delay* parameter in one of the
...
-  ```
+
+=== "Markdown" 
+   ```python
+   <|{text}|input|change_delay=800|>
+   ```
+=== "Python"
+   ```python
+   tgb.input("{text}", change_delay=800)
+   ```
```
Comment (jrobinAV) [inline docs/tutorials/articles/colab_with_ngrok/index.md:126]: Must be indented if it is part of the first bullet.

```suggestion
    === "Markdown" 
       ```python
       <|{text}|input|change_delay=800|>
       ```
    === "Python"
       ```python
       tgb.input("{text}", change_delay=800)
       ```
```

File: docs/tutorials/articles/colab_with_ngrok/index.md [hunk 4]
```diff
@@ -115,14 +115,21 @@ To address this issue, you can modify the *change_delay* parameter in one of the
...
 
 - **Globally**: To adjust the delay for all of Taipy's visual elements.
-  ```python
-  Gui(pages=pages).run(change_delay=800, ngrok_token=<Authtoken>)
-  ```
+
+```python
+Gui(pages=pages).run(change_delay=800, ngrok_token=<Authtoken>)
+```
```
Comment (jrobinAV) [inline docs/tutorials/articles/colab_with_ngrok/index.md:132]: must be indended (4 spaces) if it is part of the bullet point.

File: docs/tutorials/articles/3rd_party_components/index.md (hunk 1 above)
Comment (FlorianJacta) [reply docs/tutorials/articles/3rd_party_components/index.md:108]: We didn't have this discussion. If we do and proceed with putting Python first, it should be for every code

File: docs/tutorials/articles/3rd_party_components/src/example.py (hunk 2 above)
Comment (FlorianJacta) [reply docs/tutorials/articles/3rd_party_components/src/example.py:50]: This is done to have TGB and Markdown here

File: docs/tutorials/articles/colab_with_ngrok/index.md (hunk 3 above)
Comment (FlorianJacta) [reply docs/tutorials/articles/colab_with_ngrok/index.md:126]: Will this work? I don(t know what is the difference

File: docs/tutorials/articles/colab_with_ngrok/index.md (hunk 4 above)
Comment (FlorianJacta) [reply docs/tutorials/articles/colab_with_ngrok/index.md:132]: Same thing. I didn't know this was possible

**Comment by FlorianJacta** [comment]
@jrobinAV I was waiting for the merge of #1105. Then, I wanted to created another PR to merge with develop

File: docs/tutorials/articles/colab_with_ngrok/index.md (hunk 3 above)
Comment (jrobinAV) [reply docs/tutorials/articles/colab_with_ngrok/index.md:126]: It should

**Review by jrobinAV — APPROVED** [review]

**Comment by jrobinAV** [comment]
> @jrobinAV I was waiting for the merge of #1105. Then, I wanted to created another PR to merge with develop

Sounds fair !!

**Pull request merged**